awsutil/awsutil_test.go

```diff
@@ -0,0 +1,47 @@
+package awsutil
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestECSTaskMeta(t *testing.T) {
+	cases := []struct {
+		name         string
+		env          map[string]string
+		expectRegion string
+	}{
+		{
+			name:         "no-env",
+			env:          nil,
+			expectRegion: "bogus-east-1",
+		},
+		{
+			name:         "aws-region",
+			env:          map[string]string{"AWS_REGION": "region-from-aws-region"},
+			expectRegion: "region-from-aws-region",
+		},
+		{
+			name:         "aws-default-region",
+			env:          map[string]string{"AWS_DEFAULT_REGION": "region-from-aws-default-region"},
+			expectRegion: "region-from-aws-default-region",
+		},
+	}
+
+	for _, c := range cases {
+		os.Clearenv()
+		for k, v := range c.env {
+			require.NoError(t, os.Setenv(k, v))
+		}
+
+		ecsMeta := ECSTaskMeta{
```

Comment:
nit: if this struct was declared at the beginning of the test function then it would make the first test case a bit clearer because I could see where `bogus-east-1` is coming from.